.buildkite/release-automation/forge_arm64.Dockerfile

```diff
@@ -11,14 +11,28 @@ set -euo pipefail
 
 apt-get update
 apt-get upgrade -y
-apt-get install -y curl zip
+apt-get install -y -qq \
+    curl zip clang-12
+
+ln -s /usr/bin/clang-12 /usr/bin/clang
 
 # Install miniconda
 curl -sfL https://repo.anaconda.com/miniconda/Miniconda3-py38_23.1.0-1-Linux-aarch64.sh > /tmp/miniconda.sh
 bash /tmp/miniconda.sh -b -u -p /root/miniconda3
 rm /tmp/miniconda.sh
 /root/miniconda3/bin/conda init bash
 
+# Install Bazelisk
+curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.19.0/bazelisk-linux-arm64 --output bazelisk
+chmod +x bazelisk
```

Comment:
this is running as root here, so it can just directly save it under `/usr/bin`

Reply:
Fixed! This is saved in `/usr/loca/bin` directly now